### PR TITLE
Update collection slugs similarly to project slugs

### PR DESCRIPTION
**Pull request by edpaget**
Add a rake task to update the slugs in Jenkins

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@
   enable_extension "plpgsql"
   enable_extension "intarray"
   enable_extension "pg_trgm"
+  enable_extension "hstore"
```
Comment (camallen) [inline db/schema.rb]: these extensions crept in again..

File: spec/support/slugged_name.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+RSpec.shared_examples "has slugged name" do
+  describe '#slugged_name' do
+    let(:owner){ create :user, login: 'somebody' }
```
Comment (camallen) [inline spec/support/slugged_name.rb]: do you want to spec the case where an owner is a user group?

File: db/schema.rb (hunk 1 above)
Comment (edpaget) [inline db/schema.rb]: Everytime I migrate without dropping first...

Comment (camallen) [inline db/schema.rb]: yeah i normally do a `rake db:migrate:reset RAILS_ENV=true` but it'd be nice to not have to worry about this. I think it's cause the extensions exist on the staging db (perhaps an old migration) so the schema dumper reads it and adds it to the the dump.

**Pull request merged**